src/components/views/styl/ExperimentsTable.styl

```diff
@@ -8,21 +8,19 @@
     a, a:visited {
         color: #00f;
         text-decoration: none;
+        padding: 0 10px;
     }
 
-    thead {
-        border-bottom: 3px solid #ccc;
+    .experiment-creation-date {
+        width: 20%;
     }
 
-    tbody tr {
-        border-bottom: 1px solid #ccc;
-
-        &:last-child {
-            border-bottom: 0;
-        }
+    tr {
+        padding: 10px 0 9px;
```

Comment:
For some reason this rule doesn't appear to be having any effect. Try disabling it and notice that it looks the same.

Reply:
I'll remove all these and styling tables is a nightmare...there was another anchor added to this table once but it has since been removed.